src/app/shared/components/cluster-summary/component.ts

```diff
@@ -124,6 +123,14 @@ export class ClusterSummaryComponent {
     );
   }
 
+  get showIPv4(): boolean {
+    return this.isDualStackNetworkSelected && this.isIPv4NetworkConfigured;
+  }
+
+  get showIPv6(): boolean {
+    return this.isDualStackNetworkSelected && this.isIPv6NetworkConfigured;
```

Comment:
convert following get to private methods since none of them are used in template
- isDualStackNetworkSelected
- isIPv6NetworkConfigured